pkg/types/types.go

```diff
@@ -16,11 +16,11 @@
 package types
 
 import (
-	"gopkg.in/k8snetworkplumbingwg/multus-cni.v3/pkg/logging"
```

Comment:
Could you please let me know why you change the import list? If you don't have any clear reason, we'd prefer to keep as it is.

Reply:
@s1061123 The package import order is confusing. For example, `golangci-lint` has an explicit requirement for package import order. Also, the package import order here is different from the other code files. Typically, golang's built-in packages are imported first, and then third-party packages are imported

Reply:
Currently, as far as I know of, we don't have strict import rule in this repository yet. So this change should be done once we have the one. Please revert this change for now.

Reply:
Order has been preserved